keymapper/injection/global_uinputs.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+# key-mapper - GUI for device specific keyboard mappings
+# Copyright (C) 2021 sezanzeb <proxima@sezanzeb.de>
+#
+# This file is part of key-mapper.
+#
+# key-mapper is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# key-mapper is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with key-mapper.  If not, see <https://www.gnu.org/licenses/>.
+
+
+import evdev
+from evdev.ecodes import EV_KEY
+
+from keymapper.utils import DEV_NAME, is_keyboard_code
+
+
+class GlobalUInputs:
+    """Manages all uinputs that are shared between all injection processes."""
+    def __init__(self):
+        self.keyboard = None
+        self.gamepad = None
```

Comment:
I think we should internally manage the devices in a dict and not hardcode them. That way we can create new devices or remove unwanted devices during runtime.

Reply:
yes